Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM maven:3.5-jdk-7-alpine as build
+WORKDIR /app
+COPY . /src/main
+WORKDIR /src/main
+RUN mvn install
+
+# This line ensures that all the default tomcat ui and documentation is deleted
+ENV JAVA_OPTS="-Xms50M -Xmx50M"
```

Comment:
This needs to be higher. You're basically limiting the Java VM to use only 50MB `-Xmx2G` might the right value here